src/wizard/android/profiling-onboarding/android/3.configure-profiling.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Android
+doc_link: https://docs.sentry.io/platforms/android/profiling/
+support_level: production
+type: language
+---
+
+#### Configure Profiling
+
+Profiling depends on Sentry’s performance monitoring product being enabled beforehand. To enable performance monitoring in the SDK:
+
```

Comment:
Is there a typo here? Confused as to why this is the same as what's written under "Configure Performance" above?
```suggestion

Sentry’s performance monitoring product has to be enabled in order for Profiling to work. To enable performance monitoring in the SDK:

```